### PR TITLE
add docs to admob readme

### PR DESCRIPTION
**Pull request by sethladd**
makes it easier to update as the plugin is updated

### CODE REVIEW

**Comment by sethladd** [comment]
cc @HansMuller

**Comment by HansMuller** [comment]
The reason the material from https://github.com/flutter/plugins/pull/171 wasn't included in the README was that @collinjackson asked me to wait until the plugin was ready for production use before providing complete instructions.

File: packages/firebase_admob/README.md [hunk 1]
```diff
@@ -1,8 +1,67 @@
 # firebase_admob
-A Flutter plugin based on the [Firebase AdMob API](https://firebase.google.com/docs/admob/).
 
-*Warning*: This plugin is still under development, some AdMob features are not available yet and testing has been limited. [Feedback](https://github.com/flutter/flutter/issues) and [Pull Requests](https://github.com/flutter/plugins/pulls) are welcome.
+A plugin for [Flutter](https://flutter.io) that supports loading and
```
Comment (branflake2267) [inline packages/firebase_admob/README.md]: Would you like to add the admob badge? 
```
[![pub package](https://img.shields.io/pub/v/admob.svg)](https://pub.dartlang.org/packages/admob)
```

Comment (HansMuller) [inline packages/firebase_admob/README.md]: When we're worthy, then of course!

**Comment by sethladd** [comment]
What makes us "ready for production" ? If the features in here are usable today, I'd vote for adding instructions to the README. We don't have to be feature complete before being production ready, but interested in hearing if banner and interstitials are ready for users.

**Comment by HansMuller** [comment]
As far as I know the plugin is usable today, modulo the limitations you've copied into the README. 

If you're signing off on the debut then the updated README LGTM.

**Comment by sethladd** [comment]
I left the bold warning at the top :)

Will wait for Collin, but I'm comfortable with the disclaimer and the example code.

**Comment by mit-mit** [comment]
Has this actually been published yet? I could not find it in Pub.

File: packages/firebase_admob/README.md [hunk 2]
```diff
@@ -1,8 +1,67 @@
...
+
+Ads can be disposed to free up plugin resources. Disposing a banner
+ad that's been shown removes it from the screen. Interstitial ads
+can't be programatically removed from view.
+
+Ads can be created with a `MobileAdEvent` listener. The listener
+can be used to detect when the ad has actually finished loading
+(or failed to load at all).
+
+## Limitiations
```
Comment (collinjackson) [inline packages/firebase_admob/README.md]: check spelling

File: packages/firebase_admob/README.md [hunk 3]
```diff
@@ -1,8 +1,67 @@
...
+
+This is just an initial version of the plugin. There are still some limitiations:
+
+- Banner ads always appear at the bottom of the screen, they can't be positioned or animated into view.
+- It's not possible to specify a banner ad's size.
+- There's no support for rewarded video ads or native ads
+- The existing tests pretty rudimentary.
```
Comment (collinjackson) [inline packages/firebase_admob/README.md]: check grammar

**Comment by collinjackson** [comment]
Looks like it hasn't been published yet, I would be happy to do that after this PR is merged

**Comment by mit-mit** [comment]
@sethladd @collinjackson can we merge this, and get the package published?

**Comment by sethladd** [comment]
publishing sgtm

**Comment by collinjackson** [comment]
Seth can you please address the (trivial) review comments above and merge this PR? Happy to hit publish once that's done.

**Comment by sethladd** [comment]
Thanks, all set.

**Pull request merged**